bin/shodan

```diff
@@ -1070,6 +1077,13 @@ def stream(color, fields, separator, limit, datadir, ports, quiet, timeout, stre
 
     if datadir:
         fout = open_file(datadir, last_time)
+        fout.write('[\n')
```

Comment:
I like the idea of a --strict-json so it's an option for users that need it, though it wouldn't be the default due to the format-breaking issues.